spec/models/invitation_spec.rb

```diff
@@ -75,4 +75,41 @@
       end
     end
   end
+
+  describe "can_resend?" do
```

Comment:
Per https://www.betterspecs.org/ (under "Describe Your Methods"), the recommended naming for methods starts with `#`:
```suggestion
  describe "#can_resend?" do
```